test-support/page-object/properties/collection.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+import Ceibo from 'ceibo';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["Ceibo" is defined but never used](https://www.codacy.com/app/san650/ember-cli-page-object/file/1957769446/issues/source?bid=2903707&fileBranchId=3076256#l2)